merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/types/Duration.java

```diff
@@ -95,16 +96,7 @@
  * within your own domain in terms of the precise units we give here.
  * </p>
  */
-public final class Duration implements Comparable<Duration> {
-  // Range of (-2^63) to (2^63 - 1) microseconds.
-  // This comes out to almost 600,000 years, at microsecond resolution.
-  // Merlin was not designed for time scales longer than this.
-  private final long durationInMicroseconds;
-
-  private Duration(final long durationInMicroseconds) {
-    this.durationInMicroseconds = durationInMicroseconds;
-  }
-
+public record Duration(long micros) implements Comparable<Duration> {
```

Comment:
Yes. Unfortunately goal authors now need to directly interact with `SerializedValue` to construct activity arguments. If the activity takes a duration, they need to provide it in microseconds. They could write something like `SerializedValue.of(Duration.hours(2).dividedBy(Duration.MICROSECOND))`, but I think it could get confusing remembering which unit it should be (seconds would be reasonable too). So instead I'd rather they get in the habit of writing `SerializedValue.of(Duration.hours(2).micros())`. Since it is the only getter of its kind (there is no `.seconds()` I'm hoping it'll be clearer that `micros()` is a little more fundamental than other units.